<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceUpdates::CSettings</class>
 <widget class="QWidget" name="ServiceUpdates::CSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="CParametersWidget" name="parametersPanel" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="Settings::CLanguageWidget" name="pageLanguageSettings"/>
     <widget class="Settings::CDateTimeWidget" name="pageDateTimeSettings"/>
     <widget class="CNetworkWidget" name="pageNetworkSettings"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CParametersWidget</class>
   <extends>QWidget</extends>
   <header>ServiceUpdates/Include/ParametersWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings::CDateTimeWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/DateTimeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings::CLanguageWidget</class>
   <extends>QWidget</extends>
   <header>Settings/Include/LanguageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CNetworkWidget</class>
   <extends>QWidget</extends>
   <header>ServiceUpdates/Include/NetworkWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
